PLUB/Sources/Views/Home/ViewModel/DetailRecruitmentViewModel.swift

```diff
@@ -123,6 +122,11 @@ final class DetailRecruitmentViewModel: DetailRecruitmentViewModelType {
     }
     .asDriver(onErrorDriveWith: .empty())
     
+    categories = successFetchingDetail.map { response -> [String] in
+      return response.categories
+    }
+    .asDriver(onErrorDriveWith: .empty())
```

Comment:
keyPath를 이용하면 한 줄로 끝낼 수 있을 것 같아요.
```suggestion
    categories = successFetchingDetail.map(\.categories).asDriver(onErrorDriveWith: .empty())
```

Reply:
수정해서 올리도록 하겠습니다 :)